source/world/entity/EntityType.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+enum EntityType
+{
+	ENTITY_TYPE_NONE = 0,
+
+	// Animals
+	ENTITY_TYPE_CHICKEN = 10,
+	ENTITY_TYPE_COW,
+	ENTITY_TYPE_PIG,
+	ENTITY_TYPE_SHEEP,
+
+	// Monsters
+	ENTITY_TYPE_ZOMBIE = 32,
+	ENTITY_TYPE_CREEPER,
+	ENTITY_TYPE_SKELETON,
+	ENTITY_TYPE_SPIDER,
+	ENTITY_TYPE_PIG_ZOMBIE,
+
+	// Non-living
+	ENTITY_TYPE_ITEM = 64
```

Comment:
Question, do these match anything official?

Reply:
Are you referring to the values, or their naming? If you're referring to their values, then yes, they're from 0.7.0.